lib/tasks/whitehall_news_importer.rb

```diff
@@ -31,6 +31,8 @@ def create_or_update_document(translation, edition, document)
         review_state: "unreviewed",
         summary: translation["summary"],
         tags: tags(edition),
+        has_live_version_on_govuk: edition["state"] == "published",
+        update_type: "major",
```

Comment:
Does this do any filtering? I was expecting a next if doc_type == "government_response" or similar but not sure if I've missed something.

Reply:
This was one of the other (smaller) fixes I found while investigating the problem. It'll make sure we're showing change notes consistently with how they appear in Whitehall. Again, happy to split into a separate PR if you want :-).

Reply:
But you're right, somehow the main filtering change got lost from the commit - it's in there now (import.rake).